include/elf_parser.hpp

```diff
@@ -0,0 +1,191 @@
+#pragma once
+#include <bitset>
+#include <err.h>
+#include <fcntl.h>
+#include <gelf.h>
+#include <libelf.h>
+#include <print>
+#include <stdlib.h>
+#include <string>
+#include <sysexits.h>
+#include <unistd.h>
```

Comment:
```suggestion
#pragma once

#include <err.h>
#include <fcntl.h>
#include <stdlib.h>
#include <sysexits.h>
#include <unistd.h>

#include <gelf.h>
#include <libelf.h>

#include <bitset>
#include <string>
#include <print>
```

Lets follow the libhal style guide for headers: https://libhal.github.io/4.1/contributor_guide/style/#s143-include-order